articles/digital-assistants-vs-digital-companions/index.md

```diff
@@ -0,0 +1,63 @@
+
+1. Artificial Intelligence ([IA](https://www.section.io/engineering-education/artificial-intelligence-future/)) – Simulation of human intelligence through programming in a machine to mimic human actions.
+2. Natural Language Processing ([NLP](https://machinelearningmastery.com/natural-language-processing/)) – refers to automatic manipulation of a natural language such as speech and text through software.
+3. [Machine Language](https://www.computerhope.com/jargon/m/machlang.htm) – refers to a collection of binary digits or bits that a machine can understand.
+
+Just as the word digital assistant implies, their primary purpose is to assist us in our daily tasks, thus providing short-cuts, quick information, automate, and simplify routines.
+
+We use digital assistants in various forms, like in our smartphones, speakers, etcetera. Most of us have a fair share of interaction with digital assistants in various shapes and forms. The interaction of digital assistant takes place through hands-free [Voice User Interfaces](https://www.interaction-design.org/literature/topics/voice-user-interfaces) (a system that allows users to interact through voice or speech commands) and text or visuals. Examples of digital assistants we use today are plus [Siri](https://www.apple.com/siri/), [Alexa](https://www.cnet.com/news/what-is-alexa/), and Google.
+
+Digital assistants are affordable, convenient, and user-friendly and are primarily used to save time and carry commands. They respond to our requests reactively rather than engaging us proactively.
+
+### An overview of digital companions
+
+Digital companions are just new and improved digital assistants. In a way, they assist us in numerous tasks, and they do so in a personalized, seamless, and fun manner. They are interactive to the user. Digital companions learn and remember what we like or dislike each time we interact, that is individual preferences and needs. They also assist us in understanding and choosing, among many options, the best course of action. In other words, digital companions interact with us proactively as opposed to digital assistants, which do so reactively.
+
+Digital companions have a two-way relationship with their users based on empathy, trust, and anticipation of needs and preferences. They make suggestions that fit our interests and positively influence our behaviors.
+
+Digital companions are used to automating management and employee experience, thereby making it more accessible. It is also used in performing common tasks such as resetting passwords, system status updates, and outage alerts.
+
+### A comparison of digital assistants and digital companions
+
+- Digital assistants&#39; interactions are primarily command-based, but those of digital companions are both proactive and command-based.
```

Comment:
Can you create a markdown table for this? Check this [link](https://www.makeuseof.com/tag/create-markdown-table/). A table would make it easier for the reader to compare the two.